docs/content/2.components/1.nuxt-img.md

```diff
@@ -26,7 +26,7 @@ Will result in:
 ```
 
 ::alert{type="info"}
-With [default provider](/getting-started/providers#default-provider), you should put `/nuxt-icon.png` inside `static/` directory to make the above example work.
+With [default provider](/getting-started/providers#default-provider), you should put `/nuxt-icon.png` inside `public/` directory for Nuxt3 and in `static/` for Nuxt2 to make the above example work.
```

Comment:
Since these docs are targetting Nuxt 3, it is easier to simply mention Nuxt 3 conventions only. Renaming is part of Nuxt 2 => Nuxt 3 migration.

```suggestion
With [default provider](/getting-started/providers#default-provider), you should put `/nuxt-icon.png` inside `public/` directory for Nuxt 3 make the above example work.
```